### PR TITLE
CONSF-49 | karma-jspm implementation

### PR DESCRIPTION
**Pull request by kenkouot**
This PR adds a unit test runner implemented with karma-jspm. Actual unit tests still need to be added.

### CODE REVIEW

**Comment by lizlux** [comment]
Please add a description and update the title

File: src/core.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* global beforeEach, describe, expect, it */
```
Comment (lizlux) [inline src/core.spec.js:1]: Let's keep the tests in a `/tests` folder. I like that they are named `*.spec.js` though.

**Comment by lizlux** [comment]
Please update `package.json` so we can run `npm test`

File: src/core.spec.js (hunk 1 above)
Comment (katanka) [inline src/core.spec.js:1]: The decision to keep them in the same folder was made by @kenkouot

File: package.json [hunk 2]
```diff
@@ -40,6 +44,8 @@
     }
   },
   "dependencies": {
-    "handlebars": "^4.0.3"
+    "handlebars": "^4.0.3",
+    "jspm": "^0.16.12",
+    "karma": "^0.12.37"
```
Comment (lizlux) [inline package.json]: `karma` should only be in `devDependencies`

File: package.json [hunk 3]
```diff
@@ -40,6 +44,8 @@
     }
   },
   "dependencies": {
-    "handlebars": "^4.0.3"
+    "handlebars": "^4.0.3",
+    "jspm": "^0.16.12",
```
Comment (lizlux) [inline package.json]: `jspm` should only be in one place - `dependencies` or `devDependencies`

**Comment by lizlux** [comment]
I'd prefer it if the dependencies were all installed via `jspm install` so we don't have to run two commands to make sure our deps are up to date. Right now they only appear in `package.json`. @kenkouot, thoughts?

File: package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "dev": "jspm install && python -m SimpleHTTPServer",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "karma start karma.conf.js"
```
Comment (kenkouot) [inline package.json]: let's actually point that at the local karma. `node ./node_modules/karma/bin/karma start karma.conf.js`

File: src/core.spec.js (hunk 1 above)
Comment (kenkouot) [inline src/core.spec.js:1]: Yeah, it's more pod-like which is what we're moving to in other repos as well. Glob syntax makes filtering these files out on build steps really trivial and putting the tests next to their source keeps it top of mind.

**Comment by kenkouot** [comment]
@lizlux jspm isn't meant as an npm replacement, it's meant as a bower replacement. At least right now it is, so jspm will not fully replace npm. But it does replace bower.

**Pull request merged**